examples/dotenv-linter.jsonc

```diff
@@ -0,0 +1,55 @@
+{
+    "any-lint.disableConfirmToAllowToRun": true,
```

Comment:
Please do not add this setting.
This setting should be in trustworthy workspace only. (like corporate private repository)